docs/advanced/tls_porting.md

```diff
@@ -24,46 +24,56 @@ mbed TLS distinguishes between strong and weak entropy sources. Of the sources r
 
 The preferred way to provide a custom entropy source:
 
-1. Implement the `mbedtls_hardware_poll` function to let it access the device's entropy source.
-2. Set the appropriate macros for your target in `hal/targets.json`.
+1. Implement the `trng_api.h` to let mbed TLS access the device's entropy source.
```

Comment:
I would prefer something like 'Provide an implementation of the functions defined in `ting_api.h`' which I think is clearer.